docs/schema/V1/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@digdir/dialogporten-schema",
-  "version": "1.0.7",
+  "version": "1.1.0",
```

```suggestion
  "version": "1.0.7",
```

We can leave the version as is, I'm keeping track and publishing manually when changes occur until me and Are figure out an automatic process